dags/incline_dash.py

```diff
@@ -0,0 +1,64 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+
+from operators.bq_sensor import BigQuerySQLSensorOperator
+
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.gcs_to_gcs import GoogleCloudStorageToGoogleCloudStorageOperator
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 02, 27),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+
+with DAG('incline_dashboard',
+         default_args=default_args,
+         schedule_interval='@daily') as dag:
+
+    gcp_conn_id = 'google_cloud_derived_datasets',
+    fully_qualified_table_name = 'moz-fx-data-shared-prod.analysis.f2f__growth_dashboard'
+
+    # Check that we have data for this date
+    check_sql = (
+        'SELECT COUNT(*) '
+        'FROM `{}` '
+        'WHERE DATE(submission_timestamp) = "{}"'
+    ).format(fully_qualified_table_name, '{{ ds }}')
+
+    wait_for_data = BigQuerySQLSensorOperator(
+        task_id='wait_for_data',
+        sql=check_sql,
+        bigquery_conn_id=gcp_conn_id,
+        use_legacy_sql=False
+    )
+
+    gcs_bucket = 'moz-fx-data-prod-analysis'
+    incline_prefix = 'incline'
+    gcs_prefix = incline_prefix + '/latest'
+    gcs_uri = 'gs://{bucket}/{prefix}'.format(bucket=gcs_bucket, prefix=gcs_prefix)
+
+    table_extract = BigQueryToCloudStorageOperator(
+        task_id='bq_to_gcs',
+        source_project_dataset_table=fully_qualified_table_name,
+        destination_cloud_storage_uris=[gcs_uri],
+        bigquery_conn_id=gcp_conn_id,
+        export_format='CSV',
+        print_header=True
+    )
```

Comment:
Based on my reading of this, we're going to produce a single CSV file called `gs://moz-fx-data-prod-analysis/incline/latest`. I'd prefer this end with an explicit `.csv` suffix. So `incline/latest.csv` or preferred would be `incline/latest/data.csv`. Then we'd back up the data to `incline/archive/2020-02-23/data.csv`